lib/osut/version.rb

```diff
@@ -29,5 +29,5 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 module OSut
-  VERSION = "0.3.1".freeze
+  VERSION = "0.4.0".freeze
```

Comment:
Upgrading the revised version number, as changes are too significant for 3rd-party gems that rely on OSut.